example/themes/README.md

```diff
@@ -1,7 +1,19 @@
 # Themes
-Please make sure that the theme name and the file name are the same (+`.kdl`).
+
+Themes can contain different flavors in one file, or can be created as individual files.
 
 Example:
 
-- theme: gruvbox
-- filename: `gruvbox.kdl`
+```
+gruvbox.kdl
```

Comment:
This is not a format we want in the example directory.

Reply:
I apologize for doing the merge first.

At this point, I just wanted to make it clear that a single file can contain multiple themes, and I didn't think much more than that. Because there is a detailed explanation in the official documentation.

Do you think we need a more detailed explanation here?